routes/index.js

```diff
@@ -438,7 +438,8 @@ module.exports = function () {
       keywords: parsedWithFrontMatter.attributes.keywords,
       metaImage: parsedWithFrontMatter.attributes.image,
       published: published ? published.toISOString() : '',
-      modified: modified ? modified.toISOString() : ''
+      modified: modified ? modified.toISOString() : '',
+	  image: parsedWithFrontMatter.attributes.metaImage
```

Comment:
I don't think we need another property here - why can't we use existing `metaImage` property?

You need to go to awesome-data repo and rename recently added `metaImage` property to `image`.

Reply:
got it